content/6_Advanced/SPneg.mdx

```diff
@@ -393,7 +393,64 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+INF = 10**18
```

Comment:
float("inf")

Reply:
the thing is that with float('inf') it gives TLE.

TLE with float('inf'): https://open.kattis.com/submissions/14642683

AC (without float('inf'): https://open.kattis.com/submissions/14642731

Reply:
can you mention that using float(inf) gives tle

Reply:
Like adding a note to it.

Reply:
a comment

Reply:
done